tests/test_git_operations.py

```diff
@@ -45,3 +46,100 @@ def is_commit_present_in_repository(revision: AnyRevision) -> bool:
                                                       later=LocalBranchShortName('master')) is False
         assert self.repo_sandbox.is_ancestor_or_equal(earlier=LocalBranchShortName('develop'),
                                                       later=LocalBranchShortName('feature')) is True
+
+    def test_is_equivalent_tree_reachable_with_squash_merge(self) -> None:
```

Comment:
These tests are failing now as well ☹️

Reply:
Yeah, sorry about that, pushed too early. They should be working now.